Events.php

```diff
@@ -103,30 +103,30 @@ public static function onProfileHeaderControlsInit($event)
 
     public static function onRestApiAddRules()
     {
-        Yii::$app->urlManager->addRules([
+        Yii::$app->getModule('rest')->addRules([
```

Comment:
Can you please try to add something like: 
``` 
        /** @var \humhub\modules\rest\Module $module */
        $module = Yii::$app->getModule('rest');
```

Reply:
commit https://github.com/humhub/humhub-modules-mail/pull/217/commits/460e0915dad44a475603d904ffe6d12cf29ca669